torch/_inductor/codegen/cpp.py

```diff
@@ -294,7 +298,9 @@ def visit_modular_indexing(divisor, modulus):
 
 
 @functools.lru_cache
-def stride_at_vec_range(index: sympy.Expr, var: sympy.Symbol, vec_length: int):
+def stride_at_vec_range(
+    index: sympy.Expr, var: sympy.Symbol, vec_length: Optional[int] = None
+):
     index_vec_simplified = simplify_index_in_vec_range(index, var, vec_length)
```

Comment:
Suggest to check whether to call `simplify_index_in_vec_range` instead of passing `vec_length` to it.

Reply:
Modified, thanks for your suggestion!